roleutils/roles.py

```diff
@@ -193,6 +193,42 @@ async def remove(self, ctx: commands.Context, member: TouchableMember, *, role:
         await member.remove_roles(role, reason=reason)
         await ctx.send(f"Removed `{role.name}` from **{member}**.")
 
+    @commands.admin_or_permissions(manage_roles=True)
+    @commands.bot_has_permissions(manage_roles=True)
+    @_role.command()
+    async def addmulti(self, ctx: commands.Context, role: StrictRole, *members: TouchableMember):
+        """Add a role to multiple members."""
+        if not members:
+            raise commands.BadArgument
```

Comment:
consider usage of [require_var_positional=True](https://discordpy.readthedocs.io/en/latest/ext/commands/api.html?highlight=require%20var#discord.ext.commands.Command.require_var_positional) in the command decorator rather than manually raising if members are missing. i know older commands don't use this so it would be appreciated to change those too